orm.go

```diff
@@ -453,3 +469,19 @@ const (
 	TimeQuantumMonthDayHour     TimeQuantum = "MDH"
 	TimeQuantumYearMonthDayHour TimeQuantum = "YMDH"
 )
+
+type CacheType string
```

Comment:
I think that's useful as documentation at the least (didn't check whether there's some additional special IDE/tooling support for that). `FrameOptions` is defined like:
```
type FrameOptions struct {
	RowLabel string
	TimeQuantum TimeQuantum
	InverseEnabled bool
	CacheType      CacheType
	CacheSize      uint
}
```
So that immediately tells that `CacheType` field is of type `CacheType` (not just a generic string or int).

Also that serves as lightweight data encapsulation as well. If we one day change the underlying data type from string to int (long shot), the user code don't need to break.